packages/example-react-ts/.storybook/main.ts

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  framework: '@storybook/react',
+  stories: ['../stories/**/*.stories.mdx', '../stories/**/*.stories.@(js|jsx|ts|tsx)'],
+  addons: ['@storybook/addon-a11y', '@storybook/addon-links', '@storybook/addon-essentials'],
+  core: {
+    builder: 'storybook-builder-vite',
+  },
+  features: {
+    storyStoreV7: true,
```

Comment:
I'm not 100% positive we should have this turned on in this story, but I think it's fine to leave in, and we can take it out later if we want.